report_xlsx/models/header_footer.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import io
+from odoo import fields, models, api, _
+from odoo.tools.safe_eval import safe_eval
+from odoo.exceptions import ValidationError
+
+
+class ReportHeaderFooter(models.Model):
+    _name = 'report.xlsx.hf'
+
+    name = fields.Char(string="Name")
+    hf_type = fields.Selection(
+        [('header', 'Header'), ('footer', 'Footer')],
+        string="Type",
+        required=True)
+    value = fields.Char(string="Value")
+    manual_options = fields.Char(string="Options")
+    image_left = fields.Binary(string='Image left')
+    image_left_name = fields.Char('File Name')
+    image_center = fields.Binary(string='Image center')
+    image_center_name = fields.Char('File Name')
+    image_right = fields.Binary(string='Image right')
+    image_right_name = fields.Char('File Name')
+    header_report_ids = fields.One2many(
+        'ir.actions.report.xml',
+        'header_id',
+        string="Associated report(s)")
+    footer_report_ids = fields.One2many(
+        'ir.actions.report.xml',
+        'footer_id',
+        string="Associated report(s)")
+
+    @api.multi
+    @api.constrains('manual_options')
+    def _check_manual_options(self):
+        for rec in self:
+            options = safe_eval(rec.manual_options)
```

Comment:
In this case, `ast.literal_eval` is probably sufficient, since there are no context variables to evaluate. It's safer.
Also, can you handle the case where manual_options is not set? (currently it errors)